doc/tech/README.md

```diff
@@ -0,0 +1,148 @@
+For Networking, there are two common networking interfaces used: virtio-net for KVM/QEMU and VMXNET for VMware.
+
+Using a standard interface for IO means that workload doesn't need to run any proprietary software drivers for specific hardware vendors and the implementation of that workload is completely agnostics of the hardware used.
+
+**Figure 4** below shows the typical components of a para-virtualised interface: 
+- frontEnd driver: The frontEnd driver is an off-the-shelf driver that runs on the workload.
+- backEnd driver: runs on the Hypervisor and is responsible of bridging standard communications coming from applications to a hardware specific ones. 
+
+This nature of this disaggregation is what gives the para-virtualised interfaces the flexibility that makes them favourable in a virtualised environment.
+
+The downside of para-virtualisation interfaces is the involvement of the hypervisor which may introduce latency and jitter that can impact the performance.
+
+<p align="center"><img src="./figures/tech_virtio.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 4:</b> Para-Virtualised interface components (software only).</p>
+
+<a name="4.1.2"></a>
+#### 4.1.2 Direct assignment with VT-d
+
+Direct assignment (or pass-through) is when an IO device is directly assigned to a workload by-passing the hypervisor. Direct assignment is supported in intel platforms (using VT-d technology) and by AMD platforms (using AMD-V technology) as shown in **Figure 5**.
+
+Once an IO device is directly assigned to a workload, that workload will then have an exclusive access to that device and no other entities (including the hypervisor) can access it.
+
+This method provides better performance than the para-virtualised one as no hypervisor is involved but provides less flexibility and less portability.
+
+Having an IO device directly assigned to a workload means that the workload needs to run vendor specific drivers and libraries to be able to access that device which makes the workload less portable and dependent on a specific hardware type from a specific vendor which is not aligned with the overall strategy and goals of CNTT and hence this method of IO Virtualisation should not be used unless explicitly allowed as an exception as part of the transitional plan adopted by CNTT.
+
+<p align="center"><img src="./figures/tech_vtd.png" alt="virtio" title="Document Types" width="50%"/></p>
+<p align="center"><b>Figure 5:</b> Direct Assignment with VT-d.</p>
+
+<a name="4.1.3"></a>
+#### 4.1.3 Device Sharing with SR-IOV & VT-d
+
+Unlike Direct assignment, this method allows a hardware device to be shared amongst many workloads where each workload has an exclusive access to one region of the device (known as VF). 
```

Comment:
This is similar to Direct Assignment but allows the hardware devise to be shared.

Reply:
"Unlike DA..." - This isn't similar to, or unlike DA, this *is* DA. Consider: "This method partitions a hardware device into multiple regions (known as VFs), and uses Direct Assignment to give workloads exclusive access to one or more of those regions (VFs), thereby bypassing the hypervisor and simultaneously allowing multiple workloads to share the same device."